v1/ansible/utils/template.py

```diff
@@ -344,7 +343,7 @@ def template_from_string(basedir, data, vars, fail_on_undefined=False):
                 var_name = only_one.group(1)
                 if var_name in vars:
                     resolved_val = vars[var_name]
-                    if isinstance(resolved_val, (bool, Number, NoneType)):
+                    if resolved_val is None or isinstance(resolved_val, (bool, Number)):
```

Comment:
remove this file from the PR, v1 is dead code and will not be updated nor released, it is there just for quick testing.